examples/src/main/java/ExampleListVoiceCallFlow.java

```diff
@@ -0,0 +1,37 @@
+import com.messagebird.MessageBirdClient;
+import com.messagebird.MessageBirdService;
+import com.messagebird.MessageBirdServiceImpl;
+import com.messagebird.objects.MessageResponse;
+import com.messagebird.exceptions.GeneralException;
+import com.messagebird.exceptions.UnauthorizedException;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExampleListVoiceCallFlow {
+
+    public static void main(String[] args) {
+        if (args.length < 1) {
+            System.out.println("Please specify your access key, example : java -jar <this jar file> test_accesskey");
+            return;
+        }
+
+        // First create your service object
+        final MessageBirdService wsr = new MessageBirdServiceImpl(args[0]);
+
+        // Add the service to the cligient
+        final MessageBirdClient messageBirdClient = new MessageBirdClient(wsr);
+
+        try {
+            // Get list of call flows with offset and limit
+            System.out.println("Retrieving call flows list");
+            VoiceCallFlowList voiceCallFlowList = messageBirdClient.listVoiceCallFlows(0, 0);
```

Comment:
VoiceCallFlowList has to be imported, unused imports can be deleted

Reply:
reminder:VoiceCallFlowList has to be imported, unused imports can be deleted

Reply:
It was there, but I'm not sure why it wasn't visible before. Weird stuff, but now it's there